src/neo4j_graphrag/llm/mistralai_llm.py

```diff
@@ -84,10 +84,10 @@ def invoke(self, input: str) -> LLMResponse:
                 messages=self.get_messages(input),
                 **self.model_params,
             )
-            if response is None or response.choices is None or not response.choices:
+            if response is None or not response.choices:
                 content = ""
             else:
-                content = response.choices[0].message.content or ""
+                content = str(response.choices[0].message.content) or ""
```

Comment:
This is to fix this mypy error: https://github.com/neo4j/neo4j-graphrag-python/actions/runs/12086043143/job/33704535789

Reply:
I think this could hide problems if `response.choices[0].message.content` is `None` or a non-string type.

Maybe use `getattr` for more defensive programming?

```python
content = getattr(response.choices[0].message, "content", "")
if not isinstance(content, str):
    content = ""
```

If this is overkill maybe something like:

```python
message_content = response.choices[0].message.content
content: str = message_content if isinstance(message_content, str) else ""
```

Reply:
Thanks, will take a look, actually I don't know why we're running into this issue now, mistralai package was just upgraded from `1.1.0` to `1.2.3`.

Reply:
[Looks like it's because of this change.](https://github.com/mistralai/client-python/commit/0fd42415435e5c6314e252148802aed77a1d0c6a#diff-70d6ed7b626ffd0b1a78a90a45537cbc0ac7af6819822eb92ff6fd0f376074f2R34)

Do you still think it's ok to just cast to `str` here? Or should we take into `ContentChunk`?

Reply:
Resolving as we take your solution from #226